ThemeHelper.cs

```diff
@@ -62,6 +62,10 @@ static ThemeHelper()
             breadcrumb.ColorCSS = "/css/theme/colors_breadcrumb.css";
             breadcrumb.Footer = "Theme: Breadcrumb Dark, created by Breadcrumb";
             Themes.Add("breadcrumb", breadcrumb);
+            ThemeHelper baguette = Dark.MemberwiseClone() as ThemeHelper;
```

Comment:
This is a light theme apparently, so work off the light base not the dark

Reply:
should use `lightRef.MemberwiseClone()` instead or `Dark`?

Reply:
yes